names/names.py

```diff
@@ -11,12 +12,14 @@
 f.close()
 
```

Comment:
Very clean code here Michael 👍 Thanks for the heads up on which solution to check. It's awesome that you're challenging yourself and experimenting with different solutions though.